docs/src/main/asciidoc/hibernate-orm-panache-guide.adoc

```diff
@@ -333,8 +333,10 @@ Every query operation accepts passing parameters by index (`Object...`), or by n
 
 == The DAO/Repository option
 
-Look, we get it: you have a love/hate relationship with DAOs/Repositories but you can't live without them. We don't judge, we
-know life is tough and we've got you covered.
+Repository is a very popular pattern and can be very accurate for some use case, depending on
+the complexity of your needs. When your use case fits, they can save you from writing lots of boilerplate
```

Comment:
What is `they`?

Reply:
Reading from the comments, I think the first sentence was modified and the second sentence was not and is definitely not accurate anymore.